css/style.css

```diff
@@ -0,0 +1,128 @@
+@import url('https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap');
+
+html, body {
+   height: 100%; 
+}
+
+body {
+    background-color: #000;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+
+#viewport {
+    width: 640px;
+    height: 256px;
+    position: relative;
+    overflow-x: auto;
+    background-image: url("../images/water0.png");
+}
+
+#ports {
+    margin-top: 96px;
+    display: flex;
+    justify-content: space-around;
+}
+
+#ports > .port {
+    width: 64px;
+    height: 32px;
+    background-image: url("../images/port.png");
+}
+
+#ship {
+    width: 128px;
+    height: 64px;
+    background-image: url("../images/ship.png");
+    position: absolute;
+}
+
+#sailbutton {
+    border: 0;
+    background-color: #FB3640;
+    border-bottom: 4px solid #d8040f;
+    color: #FFFFFF;
+    font-family: inherit;
+    padding: 10px 20px;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    font-family: "Press Start 2P", cursive;  
+}
```

Comment:
It's generally best to add `cursor: pointer;` property to button's css, as this gives you the traditional click cursor that you get when you hover over a button to show that's it interactive.